tools/openms/CompNovo.xml

```diff
@@ -41,28 +41,28 @@ python3 '$__tool_directory__/fill_ctd.py' '@EXECUTABLE@.ctd' '$args_json' '$hard
     <configfile name="hardcoded_json"><![CDATA[{"log": "log.txt", "threads": "\${GALAXY_SLOTS:-1}", "no_progress": true}]]></configfile>
   </configfiles>
   <inputs>
-    <param name="in" argument="-in" type="data" format="mzml" optional="false" label="input file in mzML format" help=" select mzml data sets(s)"/>
+    <param argument="-in" type="data" format="mzml" optional="false" label="input file in mzML format" help=" select mzml data sets(s)"/>
     <section name="algorithm" title="Algorithm section" help="" expanded="false">
-      <param name="max_number_aa_per_decomp" argument="-algorithm:max_number_aa_per_decomp" type="integer" optional="false" value="4" label="maximal amino acid frequency per decomposition" help=""/>
-      <param name="tryptic_only" argument="-algorithm:tryptic_only" type="text" optional="false" value="true" label="if set to true only tryptic peptides are reported" help="">
+      <param argument="-algorithm:max_number_aa_per_decomp" type="integer" optional="false" value="4" label="maximal amino acid frequency per decomposition" help=""/>
+      <param argument="-algorithm:tryptic_only" type="text" optional="false" value="true" label="if set to true only tryptic peptides are reported" help="">
```

Comment:
looks like a boolean and not like a text

Reply:
https://github.com/OpenMS/OpenMS/issues/5880